mlflow/xgboost.py

```diff
@@ -227,6 +264,29 @@ def log_model(
     :param await_registration_for: Number of seconds to wait for the model version to finish
                             being created and is in ``READY`` status. By default, the function
                             waits for five minutes. Specify 0 or None to skip waiting.
+    :param pip_requirements: Either an iterable of pip requirement strings
+        (e.g. ``["scikit-learn", "-r requirements.txt"]``) or the string path to a pip requirements
+        file on the local filesystem (e.g. ``"requirements.txt"``). If provided, this describes the
+        environment this model should be run in. If ``None``, a default list of requirements is
+        inferred from the current software environment. Requirements are automatically parsed and
+        written to a ``requirements.txt`` file that is stored as part of the model. These
+        requirements are also written to the ``pip`` section of the model's conda environment
+        (``conda.yaml``) file.
+    :param additional_pip_requirements: Either an iterable of pip requirement strings
+        (e.g. ``["scikit-learn", "-r requirements.txt"]``) or the string path to a pip requirements
+        file on the local filesystem (e.g. ``"requirements.txt"``). If provided, this specifies
+        additional pip requirements that are appended to a default set of pip requirements generated
+        automatically based on the user's current software environment.
```

Comment:
Can we also include 

```
requirements are also written to the ``pip`` section of the model's conda environment (``conda.yaml``) file.
```

like we do in the docstring for `pip_requirements`?

Reply:
This looks like a dup of https://github.com/mlflow/mlflow/pull/4519#discussion_r669310782